### PR TITLE
Move Blockly.utils.dom.SvgElementType to Blockly.utils.Svg

### PR DESCRIPTION
**Pull request by samelhusseini**
## The basics
- [x] I branched from develop
- [x] My pull request is against develop
- [x] My code follows the [style guide](https://developers.google.com/blockly/guides/modify/web/style-guide)

### Resolves

Move `Blockly.utils.dom.SvgElementType` to `Blockly.utils.Svg` to shave off 2.7 KB.

### Proposed Changes

Move `Blockly.utils.dom.SvgElementType` to `Blockly.utils.Svg`. 
Add relevant requires.
Make `Blockly.utils.Svg` @package.

### Reason for Changes

Minor optimization.

### Test Coverage

Tested on:

### CODE REVIEW

File: core/utils/svg.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+/**
+ * Returns the SVG element tag name.
+ * @return {string} The name.
+ * @override
+ */
+Blockly.utils.Svg.prototype.toString = function() {
+  return this.tagName_;
+};
+
+/** @type {!Blockly.utils.Svg<!SVGAnimateElement>} */
```
[the remark this answers is not in the document]
Comment (samelhusseini) [reply core/utils/svg.js]: Done.

**Comment by rachel-fenichel** [comment]
LGTM once travis agrees

File: core/utils/svg.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
```
Comment (alschmiedt) [inline core/utils/svg.js:3]: 2020

File: core/utils/svg.js [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @fileoverview Utility methods for DOM manipulation.
+ * These methods are not specific to Blockly, and could be factored out into
```
Comment (alschmiedt) [inline core/utils/svg.js:9]: Update Comment

**Review by alschmiedt — APPROVED** [review]

**Pull request merged**